server/asset/dcr/dcr.go

```diff
@@ -261,6 +247,9 @@ func (dcr *DCRBackend) transaction(txHash *chainhash.Hash) (*Tx, error) {
 		})
 	}
 	feeRate := (sumIn - sumOut) / uint64(len(verboseTx.Hex)/2)
+	if isCoinbase {
+		feeRate = 0
```

Comment:
It's worth noting that votes with a non-zero fee aren't rejected by the network and they are generated regularly, usually with pool votes.  Depending on how this function is used it may be fine to force this to zero, but it is not necessarily accurate.